runtime/doc/digraph.txt

```diff
@@ -169,1311 +169,1312 @@ The rouble sign was added in 2014 as 0x20bd.  Vim supports the digraphs =R and
 =P for this.  Note that R= and P= are other characters.
 
 						*digraph-table* *digraph-table-mbyte*
-char  digraph	hex	dec	official name ~
-^@	NU	0x00	  0	NULL (NUL)
```

Comment:
TODO: lines following `column_heading` (`foo ~`) perhaps should be treated as true columnar data (thus in HTML rendered as `<table>`). But `column_heading` is abused in many places so would need to clean those up.